ci/scripts/shipit

```diff
@@ -73,7 +73,7 @@ echo "SHA1=$SHA1"
 export SHA256=$(sha256sum $RELEASE_TGZ | head -n1 | awk '{print $1}')
 echo "SHA256=$SHA256"
 
-PATCHED_RELEASE_TGZ=$REPO_ROOT/releases/$RELEASE_NAME/$RELEASE_NAME_patched-$VERSION.tgz
+PATCHED_RELEASE_TGZ=$REPO_ROOT/releases/$RELEASE_NAME/${RELEASE_NAME}_patched-$VERSION.tgz
```

Comment:
Did this actually work before? 😮 
Maybe wrap all variables for better readability/uniformity?

Reply:
no. That was another PR (#351) and there was no release yet with that.